Honor the `size` option after following a redirect and revert data uri support (#925)
node-fetch/node-fetch: CHANGELOG.md

# 2.x release

## v2.6.1

> **This is an important security release. We strongly encourage everyone to update as soon as possible.**

Reviewer: ```suggestion
**This is an important security release. It is strongly recommended to update as soon as possible.**
```
